client/src/test/java/io/pravega/keycloak/client/KeycloakAuthzClientTest.java

```diff
@@ -85,17 +91,32 @@ public void getRPT_error_authz() {
         when(client.authorization(any()).authorize(any())).thenThrow(new HttpResponseException("", 400, "", null));
 
         KeycloakAuthzClient authzClient = new KeycloakAuthzClient(client, tokenCache);
-        authzClient.getRPT();
+        try {
+            authzClient.getRPT();
+            Assert.fail();
+        }
+        catch(KeycloakAuthorizationException e) {
+        }
+        verify(client, times(1)).obtainAccessToken();
     }
 
-    @Test(expected = HttpResponseException.class)
+    @Test
     public void getRPT_error_other() {
         AuthzClient client = mock(AuthzClient.class, Mockito.RETURNS_DEEP_STUBS);
         TokenCache tokenCache = spy(new TokenCache(0));
         when(client.obtainAccessToken()).thenThrow(new HttpResponseException("", 500, "", null));
 
         KeycloakAuthzClient authzClient = new KeycloakAuthzClient(client, tokenCache);
-        authzClient.getRPT();
+        authzClient.setHttpMaxRetries(3);
+        authzClient.setHttpRetriesDelayMsecs(1);
+        try {
+            authzClient.getRPT();
+            Assert.fail();
+        }
+        catch(HttpResponseException e) {
+            int i = 0;
```

Comment:
What does this do? I don't see the `i` being used in any way below.

Reply:
Left over debugging code.  thanks for catching.